CHANGELOG.md

```diff
@@ -28,6 +29,198 @@
+  new `header_names` plugin configuration attribute was added that accepts
+  an array of header names the JWT plugin should use when authentication the
+  client. This defaults to `{ "Authorization" }`.
+  [#4757](https://github.com/Kong/kong/pull/4757)
+
+### Changes
+
+##### Dependencies
+
+- The minimum required OpenResty version has been bumped to `1.15.8.1`.
+  Kong's [openresty-patches](https://github.com/kong/openresty-patches)
+  must still be applied (those patches are already bundled with our official
+  distribution packages) for Kong to work properly.
+  The new OpenResty brings many new features that Kong leverages:
+  1. Kong uses the new [ngx\_http\_grpc\_module](https://nginx.org/en/docs/http/ngx_http_grpc_module.html)
+     to implement it's gRPC proxying support.
+  2. Kong have support of the new [keepalive\_requests](https://nginx.org/en/docs/http/ngx_http_upstream_module.html#keepalive_requests)
+     directive. It's default value is `100`.
+  3. Kong have support of the new [keepalive\_timeout](http://nginx.org/en/docs/http/ngx_http_upstream_module.html#keepalive_timeout)
+     directive. It's default value is `60s`.
+  4. OpenResty `1.15.8.1` added ARM64 architecture support. With it, Kong now
+     officially supports running on ARM64 as well. ARM64 artifact will be
+     part of official Kong release from now on.
+  5. OpenResty `1.15.8.1` brings better performance when handling incoming
+     requests.
+  6. The LuaJIT GC64 mode is enabled by default in OpenResty `1.15.8.1`,
+     which raised LuaJIT GC-mamaged VM limit from *2 GB* to *128 TB*.
+     In addition, GC64 mode should also result in more predictable GC
+     performance for LuaJIT.
+  [#4382](https://github.com/Kong/kong/pull/4382)
+- Bumped [kong-lapis](https://github.com/Kong/lapis) to `1.7.0.1`.
+  [#4684](https://github.com/Kong/kong/pull/4684)
+- Bumped [luaossl](https://github.com/wahern/luaossl) to `20190612`.
+  [#4735](https://github.com/Kong/kong/pull/4735)
+
+##### Plugins
+- Bumped [kong-plugin-azure-functions](https://github.com/Kong/kong-plugin-azure-functions) to `0.4`.
+  [#4699](https://github.com/Kong/kong/pull/4699)
+- Bumped [kong-plugin-kubernetes-sidecar-injector](https://github.com/Kong/kong-plugin-kubernetes-sidecar-injector) to `0.2`.
+  [#4699](https://github.com/Kong/kong/pull/4699)
+- Bumped [kong-plugin-serverless-functions](https://github.com/Kong/kong-plugin-serverless-functions) to `0.3`.
+  [#4699](https://github.com/Kong/kong/pull/4699)
+- Bumped [kong-prometheus-plugin](https://github.com/Kong/kong-prometheus-plugin) to `0.4`.
+  [#4699](https://github.com/Kong/kong/pull/4699)
+
+### Fixes
+
+##### CLI
+- Fixed an issue if admin ran `kong restart` and Kong was not running,
+  logging would be turned off.
+  [#4772](https://github.com/Kong/kong/pull/4772)
+- Fixed an issue that warning regarding Kong using deprecated config value
+  warnings being shown even when admin is stopping Kong. This ensures such
```

Comment:
"Fixed an issue **that**" is not correct English. The correct conjunction is "Fixed an issue **where**" (this happens more times in this change, below).

The bug description should be in past tense ("when admin *was* stopping kong" instead of "*is* stopping kong").

Also, repeated word: "**warning** regarding kong using deprecated config value **warnings**".

Suggested replacement:

> - Fixed an issue where warnings related with deprecated config values were shown when `kong stop` is run. The change ensures that the warnings will appear only when running `kong start` or `kong restart`.

Alternatively, just remove this item from the changelog, since it is such a minor fix.